CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix `hello_rgb` example for ESP32 (#1173)
 - Fix timer `now` for esp32c3 and esp32c6
 - Fixed the multicore critical section on Xtensa (#1175)
+- Fix timer `now` for esp32c3 and esp32c6 (#1178)
```

Comment:
```suggestion
```

Looks like this duplicated?

Reply:
oh yes this looks suspicious but that other PR didn't include the PR number in the changelog so I added it here - this is just a follow up of the first one

Reply:
Ah sorry I picked the wrong line, could you remove line 34? That's the duplicate without the PR number

Reply:
ah sure ... haven't seen it 🙈